rdrf/rdrf/templates/registration/registration_login_details.html

```diff
@@ -1,7 +1,35 @@
 {% load i18n static %}
 
+<link rel="stylesheet" href="{% static 'vendor/bootstrap-select-1.14.0-beta3/css/bootstrap-select.min.css' %}">
+
 <script type="text/javascript" src="{% static 'js/show_password.js' %}"></script>
+<script id="id_bootstrap_selector_js" type="text/javascript" src="{% static 'vendor/bootstrap-select-1.14.0-beta3/js/bootstrap-select.min.js' %}"></script>
+
 <script>
+    const scriptTag = document.createElement('script');
+    scriptTag.type = "text/javascript";
+    scriptTag.setAttribute("async", "false");
+    scriptTag.src = `/static/vendor/bootstrap-select-1.14.0-beta3/js/i18n/defaults-${navigator.languages[0].replace('-', '_')}.min.js`;
+    document.getElementById('id_bootstrap_selector_js').after(scriptTag);
+
+    function getPureLanguageCode(language) {
```

Comment:
nit: nest this within `constructPreferredLanguages` if not used from anywhere else. Could be an arrow function e.g. `const getPureLanguageCode = (language) => language.split('-')[0];`